options/posix/generic/posix_stdlib.cpp

```diff
@@ -487,3 +488,8 @@ char *canonicalize_file_name(const char *) {
 	__ensure(!"Not implemented");
 	__builtin_unreachable();
 }
+
+int clearenv(void) {
```

Comment:
This should live with the other environment functions.

Reply:
`clearenv` should probably only be included in the posix option, and even there it's kinda doubtful - it hasn't been accepted into the standard as far as I can tell. So I'm only moving this up to be right below `secure_getenv`.

Reply:
The other functions in environment.cpp are behind a posix ifdef already, so IMO it'd make more sense to go there.